sympy/stats/crv_types.py

```diff
@@ -922,7 +922,7 @@ def Exponential(name, rate):
     Parameters
     ==========
 
-    rate : A positive Real number, `\lambda > 0`, the rate (or inverse scale/inverse mean)
+    lambda : A positive Real number, `\lambda > 0`, the rate (or inverse scale/inverse mean)
```

Comment:
`lambda` is a keyword. It cannot be used as an identifier.

Reply:
Actually in https://docs.sympy.org/latest/modules/stats.html?highlight=stats#module-sympy.stats.crv_types 
for every distribution, parameters are defined in the way  `Symbol : Property/ies Name`

Eg.
https://github.com/sympy/sympy/blob/d2fcf9ecdfef464b26f0e118018a7817059d22a2/sympy/stats/crv_types.py#L271-L273
https://github.com/sympy/sympy/blob/d2fcf9ecdfef464b26f0e118018a7817059d22a2/sympy/stats/crv_types.py#L357-L358
https://github.com/sympy/sympy/blob/d2fcf9ecdfef464b26f0e118018a7817059d22a2/sympy/stats/crv_types.py#L502-L503
The symbols are in accordance with the distribution function written. I have made this change in that manner only.